File: csharp/BCUR/BCUR/BytewordsStyle.cs
namespace BlockchainCommons.BCUR;

/// <summary>
/// The three different bytewords encoding styles.
/// </summary>
public enum BytewordsStyle
{
    /// <summary>Four-letter words, separated by spaces.</summary>
    Standard,
    /// <summary>Four-letter words, separated by dashes.</summary>
    Uri,
    /// <summary>Two-letter words (first and last letter), concatenated without separators.</summary>
    Minimal
}

File: kotlin/bc-ur/src/main/kotlin/com/blockchaincommons/bcur/BytewordsStyle.kt
package com.blockchaincommons.bcur

/** The three bytewords encoding styles. */
enum class BytewordsStyle {
    /** Four-letter words, separated by spaces. */
    Standard,
    /** Four-letter words, separated by dashes. */
    Uri,
    /** Two-letter words, concatenated without separators. */
    Minimal
}
